l1-contracts/bootstrap.sh

```diff
@@ -18,8 +18,12 @@ fi
 # Clean
 rm -rf broadcast cache out serve
 
-# Install foundry.
-. ./scripts/install_foundry.sh
```

Comment:
My understanding was that this script was to install foundry into a very specific location within the monorepo.
It shouldn't be updating any global installs on the users system. And ensure we have the right version of foundry (which stung many times).
I think your check here is just looking to see if theres a global install?
Note the script installs it into the `$PWD/.foundry`

Reply:
Good point. Serves me well for trying to sneak in unrelated changes in the PR. Rolling this one back as well, we can tackle it separately.